pandas/io/parquet.py

```diff
@@ -206,12 +213,16 @@ def __init__(self):
             )
         self.api = fastparquet
 
-    def write(self, df, path, compression='snappy', index=None, **kwargs):
+    def write(self, df, path, compression='snappy', index=None,
+              partition_cols=None, **kwargs):
         self.validate_dataframe(df)
         # thriftpy/protocol/compact.py:339:
         # DeprecationWarning: tostring() is deprecated.
         # Use tobytes() instead.
 
+        if partition_cols is not None:
+            kwargs['file_scheme'] = 'hive'
```

Comment:
is this not defaulted pyarrow? if you don't specify what is the default?

Reply:
this is for fastparquet. Default file_scheme is simple. In simple _metadata file will not be generated. _metadata file is necessary to perform a read on a directory in fastparquet

Reply:
To be specific, simple means one file only, and is not appropriate for dask. There is also Drill scheme, where the directory names are different from the Hive variant (*if* using directory partitioning, no difference otherwise). Spark defaults to the Hive format, it's a reasonable default in every case.

Reply:
(the _metadata file is longer necessary for the fastparquet branch, actually, but it is preferred)

Reply:
can you add a comment above this describing what you said above, e.g. for compat with fp

Reply:
Need a comment here sayings what's going on.

Reply:
That is expected in 0.1.6, you would need to pass the whole list of data files rather than the root directory. This may be fixed in master.